### DISCUSSION TITLE
Local cache quota and Clear cache option in iPadOS app

### DISCUSSION THREAD

**Discussion by apomortsev** [immich-app/immich#9572] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Currently, the iPadOS app seems to locally cache as much data as it can until device storage is exhausted. I'm seeing this on an iPad Air 5-gen 64GB, iPadOS 17.4.1. There is no setting to control the cache size or an option to clear cache so re-installing the app seems to be the only way to free up the storage. It would be good to be able to set a cache quota and have an option to clear the cache. 

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#4560
